src/com/company/task3/MovieProcessing.java

```diff
@@ -0,0 +1,53 @@
+package com.company.task3;
+
+import com.company.task3.move.Movie;
+import com.company.task3.move.MovieGenerator;
+import com.company.task3.move.utilit.Genre;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class MovieProcessing {
+    public static void processingMovies() {
+        List<Movie> first = MovieGenerator.generateMoves(50);
+        List<Movie> second = MovieGenerator.generateMoves(50);
+
+        Map<String, List<Movie>> movies = Stream.concat(
+                first.stream(),
+                second.stream())
+                .collect(Collectors.groupingBy(Movie::getGenre));
+
+        double actionAveragePrice = movies.get(Genre.getGenre(0)).stream()
```

Comment:
create a separate method for this and reuse it

Reply:
fixed